deepchem/feat/base_classes.py

```diff
@@ -421,3 +421,23 @@ class UserDefinedFeaturizer(Featurizer):
   def __init__(self, feature_fields):
     """Creates user-defined-featurizer."""
     self.feature_fields = feature_fields
+
+
+class DummyFeaturizer(Featurizer):
+  """Abstract class for plain featurization."""
```

Comment:
Can you add a usage example here?

Reply:
Also, this isn't an abstract class since it has a concrete implementation. Can you expand the docstring to explain why we'd want a no-op featurizer like this and when users should think of using it?

Reply:
will do!